functions/__z.fish

```diff
@@ -164,9 +164,14 @@ function __z -d "Jump to a recent directory."
         if set -q _flag_echo
             printf "%s\n" "$target"
         else if set -q _flag_directory
-            type -q xdg-open;and xdg-open "$target"; and return $status;
-            type -q open;and open "$target"; and return $status;
-            echo "Not sure how to open file manager"; and return 1;
+            if test "$OS" = Windows_NT
+                type -q explorer;and explorer "$target"; return $status;
+                # Be careful, in msys2, explorer always return 1
```

Comment:
I think @jethrokuan [said](https://github.com/jethrokuan/z/pull/76/files#r263267402) that we should return 0 here.

Reply:
Oh, @jethrokuan, so which one do you expect?

Reply:
let's keep it at 0 to be standard.

Reply:
OK.